net.rpcs3.RPCS3.yaml

```diff
@@ -82,6 +82,7 @@ modules:
       - -DUSE_SYSTEM_LIBPNG=ON
       - -DUSE_SYSTEM_ZLIB=ON
       - -DITTAPI_SOURCE_DIR=/run/build/rpcs3/ittapi
+      - -DITTAPI_GIT_TAG=HEAD
```

Comment:
What is this env var for ?

Reply:
It was for a test that didn't fixed it, I forgot to remove it